pilot-app/src/components/system/channel-selection.vue

```diff
@@ -0,0 +1,47 @@
+﻿<template>
+    <div class="channel">
+        <h3>Channel Selection</h3>
+        <table>
+            <tbody>
+                <tr>
+                    <th>Channel Selection</th>
+                    <td>
+                        <wimChoiceDropdown :field="field" 
+                                           @onchange="change"></wimChoiceDropdown>
```

Comment:
Codacy has a fix for the issue: [Expected 1 line break before closing bracket, but no line breaks found.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                                           @onchange="change
```